elements/pfe-styles/src/pfe-base.scss

```diff
@@ -82,33 +82,30 @@ th {
 // }
 
 body {
-  font-family: pfe-var(font-family);
-  font-size:   var( --pf-c--FontSize, pfe-var(content--FontSize));
-  font-weight: pfe-var(font-weight--light);
-  line-height: pfe-var(line-height);
+  @extend %pfe-content-base;
+  font-size: var( --pf-c--FontSize, pfe-var(content--FontSize));
```

Comment:
Maybe add a comment here that this is the empty hook (I could see people referencing the CSS file and trying to debug something)

Reply:
👍